CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+- Bug fixes
+- New updates
+- Performance improvements
+
+The changelog entry is **not required** for
+- Any docs-only (e.g., README.md) changes
+- Any developer-facing change (e.g., test suite changes)
+
+### Writing changelog entries
+
+```
+title: 'Fix bug casting spells on magicians'
+category: fixed
+author: John Smith <jsmith@example.com>
+issue: 666
+notes: >
+    The bug was making impossible to cast a spell on
+    a magician.
+```
+
+The `issue` value is a reference to the issue, if any, that is targeted with 
+this change, and the `author` key (format: `Your Name <YourName@example.org>`) 
+is used to give attribution to community contributors. **Both are optional.** 
+The `category` field maps the category of the change, valid options are: 
+added, fixed, changed, deprecated, removed, security, performance, other. 
+**Category field is mandatory.** The `notes` field should have a description 
+explaining the changes in the code. Remember you can write blocks of text using 
+the `>` character at the beginning of each block. See the above given example.
+
+A good changelog entry should be descriptive and concise. It should explain the 
+change to a reader who has *zero context* about the change. If you have trouble 
+making it both concise and descriptive, err on the side of descriptive.
+
+- **Bad:** Go to a project order.
+- **Good:** Show a user’s starred projects at the top of the “Go to project” dropdown. 
+
+The first example provides no context of where the change was made, or why, or 
+how it benefits the user. 
+
+- **Bad:** Copy (some text) to clipboard.
+- **Good:** Update the “Copy to clipboard” tooltip to indicate what’s being copied. 
+
+Again, the first example is too vague and provides no context.
+
+- **Bad:** Fixes and Improves CSS and HTML problems in mini pipeline graph and builds dropdown.
+- **Good:** Fix tooltips and hover states in mini pipeline graph and builds dropdown. 
+
+The first example is too focused on implementation details. The user doesn’t care 
+that we changed CSS and HTML, they care about the result of those changes.
+
+- **Bad:** Strip out `nil`s in the Array of Commit objects returned from `find_commits_by_message_with_elastic`
+- **Good:** Fix 500 errors caused by Elasticsearch results referencing garbage-collected commits 
+
+The first example focuses on *how* we fixed something, not on *what* it fixes. 
+The rewritten version clearly describes the *end benefit* to the user 
+(fewer 500 errors), and *when* (searching commits with Elasticsearch).
+
+Use your best judgment and try to put yourself in the mindset of someone reading 
+the compiled changelog. Does this entry add value? Does it offer context about 
+*what* and *why* the change was made?
+
+Contributors can use the interactive [changelog](https://github.com/Bitergia/release-tools#changelog) 
+tool for this purpose which generates the changelog entry file automatically.
```

Comment:
I think this part is more useful if it goes after the description of the fields.